internal/langserver/handlers/service.go

```diff
@@ -447,6 +447,11 @@ func (svc *service) configureSessionDependencies(ctx context.Context, cfgOpts *s
 				refreshModuleProviders(svc.sessCtx, svc.server, svc.logger, commandId))
 		}
 
+		if commandId, ok := lsp.ExperimentalClientCapabilities(cc.Experimental).RefreshModuleCallsCommandId(); ok {
+			svc.stateStore.Modules.ChangeHooks = append(svc.stateStore.Modules.ChangeHooks,
+				refreshModuleProviders(svc.sessCtx, svc.server, svc.logger, commandId))
```

Comment:
I think we should either rename/repurpose this function as a generic `callClientCommandId()` or add a new one similar to `refreshModuleProviders`. Probably the former.